proposals/csharp-12.0/collection-expressions.md

```diff
@@ -129,7 +129,7 @@ The following implicit *collection expression conversions* exist from a collecti
 In the cases above, a collection expression *element* `Ei` is considered to have an *implicit conversion* to *type* `T` if:
 
 * `Ei` is an *expression element* and there is an implicit conversion from `Ei` to `T`.
-* `Ei` is a *spread element* `Si` and there is an implicit conversion from the *iteration type* of `Si` to `T`.
+* `Ei` is a *spread element* and there is an implicit conversion from the *iteration type* of the spread element *collection* to `T`.
```

Comment:
The term 'collection' surprises me a bit since any enumerable can be spread, and not all enumerables are collections.